tasks/create_app_config.yml

```diff
@@ -14,6 +14,11 @@
   set_fact:
     app_configmaps: "{{ app | json_query('services[?configs].{name: name, configs: configs}') }}"
 
+- include_tasks: tasks/run_tasks_for_app.yml
+  vars:
+    tasks:
+      - tasks/get_vault_for_app
```

Comment:
This cannot work, as we (or more precisely the deployment bot) does not have access to the vault during the deployment.